.travis.yml

```diff
@@ -3,17 +3,18 @@ go:
   - 1.19.x
 services:
   - docker
+before_script: curl -sfL https://github.com/goreleaser/goreleaser/releases/download/v1.19.1/goreleaser_Linux_x86_64.tar.gz | tar xzvf - goreleaser
```

Comment:
I'm not familiar with goreleaser. I assume that adding the `goreleaser` binary to the root of the repo won't change anything in the generated artifacts. Is that correct?

Reply:
Right, it creates a folder `dist`for that.